.rubocop.yml

```diff
@@ -1,4 +1,5 @@
 AllCops:
+  SuggestExtensions: false
```

Comment:
What is this for?

Reply:
This line allows you to auto-indent the code in the VS code.